include/xeus/xinterpreter.hpp

```diff
@@ -66,6 +66,13 @@ namespace xeus
         void display_data(nl::json data, nl::json metadata, nl::json transient);
         void update_display_data(nl::json data, nl::json metadata, nl::json transient);
         void publish_execution_input(const std::string& code, int execution_count);
+        nl::json publish_execution_output(int execution_count,
+                                      const std::string& status,
+                                      const std::string& ename,
```

Comment:
It's not always the case that a `reply` contains an error name, so maybe this should be optional, or instead we could have two methods: `make_reply` and `make_error_reply`?

Reply:
Yeah, you're right! About these things. I'll make these changes :)
Thanks.

Reply:
Another idea for a name could be `create_reply`?